src/NosCore.Packets/ClientPackets/WhisperPacket.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NosCore.Core.Serializing;
+
+namespace NosCore.Packets.ClientPackets
+{
+    [PacketHeader("/")]
+    public class WhisperPacket : PacketDefinition
+    {
+        [PacketIndex(0, SerializeToEnd = true)]
```

Comment:
character name is the first one you can have string characterName and Message serialize to end

Reply:
except when gm wont do it